.jenkins/release.Jenkinsfile

```diff
@@ -3,8 +3,11 @@ def releaseVersion
 def releaseUrl
 
 pipeline {
-    libraries { lib 'adverity-shared-library@2.6.0' }
+    libraries { lib 'adverity-shared-library@2.10.0' }
     agent { label "PRJob" }
+    options {
+        withAWS(role: 'CI', roleAccount: '221160807535')
```

Comment:
can we store those accountIds globally maybe?

Reply:
there are 2 options that come into my mind

either as global environment variables

or we actually export some constants from adverity-shared-library

Reply:
I think 1) is a good approach now, if the ids would be in the jenkins-jobs repo I wouldn't mind as finding/changing is not a problem with a global search/replace but if this is scattered across all various repos it might get a pain.
We don't have to do it now but think about it with next release setups like that.